es-ds-components/components/es-popover.vue

```diff
@@ -0,0 +1,157 @@
+
+<template>
+    <OverlayPanel ref="op" :dismissable="true" appendTo="body" :pt="popoverPt" :style="overlayPanelStyle">
```

Comment:
Although we don't have linting in place yet, I think we'll likely want to stick with kebab-case for component naming. I'd suggest switching this to `overlay-panel`